pom.xml

```
@@ -51,8 +51,8 @@
         <jboss.snapshots.repo.id>jboss-snapshots-repository</jboss.snapshots.repo.id>
         <jboss.snapshots.repo.url>https://s01.oss.sonatype.org/content/repositories/snapshots/</jboss.snapshots.repo.url>
 
-        <quarkus.version>3.8.5</quarkus.version>
-        <quarkus.build.version>3.8.5</quarkus.build.version>
+        <quarkus.version>3.13.1</quarkus.version>
+        <quarkus.build.version>3.13.1</quarkus.build.version>
```

```suggestion
        <quarkus.version>3.13.2</quarkus.version>
        <quarkus.build.version>3.13.2</quarkus.build.version>
```
3.13.2 should be already out: https://mvnrepository.com/artifact/io.quarkus.platform/quarkus-bom/3.13.2